app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to @user
```

Comment:
- [ ] [required] It should `redirect_to edit_user_path(<USER_ID>)` if user was saved (remember to change <USER_ID> to the code that works for the student implementation)